.drone.star

```diff
@@ -167,10 +167,7 @@ def main(ctx):
   return pipelines
 
 def getTestPipelinesNames(ctx):
-  pipelines = []
-
-  for module in config['modules']:
-    pipelines.append(testing(ctx, module))
+  pipelines = getTestSuiteNames()
```

Comment:
```suggestion
  pipelineNames = getTestSuiteNames()
```

These are really just pipeline names, aren't they? It is just a list of strings.

At first I got a bit confused when I wondered why a "pipeline" was just a string.